src/components/card/new-vertical-resource-card.tsx

```diff
@@ -34,8 +34,20 @@ const VerticalResourceCard: React.FC<{
   if (isEmpty(resource)) return null
 
   return (
-    <ResourceLink path={resource.path} location={location}>
+    <ResourceLink
+      path={(resource.path || resource.url) as string}
+      location={location}
+    >
       <Card {...props} resource={resource} className={className}>
+        {resource.background && (
```

Comment:
vertical cards can have a background image now. it can get tricky because of color modes, but I kept it simple for now.